### ISSUE TITLE
Restart Training?

### ISSUE THREAD

**Issue by toshi2k2** [leoxiaobin/deep-high-resolution-net.pytorch#124]
@leoxiaobin Can the details (Adam optimizer configuration, batchnorm stats) be released so that we can restart training from the saved models? It’s frankly impossible to retrain the model from scratch with ordinary computation power.
